### PR TITLE
release: highlights of v0.3.0

### PR DESCRIPTION
**Pull request by aslakknutsen**

### CODE REVIEW

File: docs/modules/ROOT/pages/release_notes/v0.3.0.adoc [hunk 1]
```diff
@@ -0,0 +1,24 @@
+== Highlights of v0.3.0 release
+
+The main focus of this release has been some improved error handling and bug fixes.
```
Comment (bartoszmajsak) [inline docs/modules/ROOT/pages/release_notes/v0.3.0.adoc:3]:
```suggestion
The main focus of this release has been to improve error handling and bug fixes.
```

File: docs/modules/ROOT/pages/release_notes/v0.3.0.adoc [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+support the header matches. Now this is changed so a connected service supports both, allowing you to `develop` exposed and internal services.
+
+=== The Session Name is now validated to follow RFC 1123
+
+The session name is used as part of the sub domains that are created and needs to follow
+the DNS naming spec. We now validate that it does before failing later. :)
+
+=== Changed to use emperror for more detailed errors
+
+With emperror we've got the ability to attach contextual metadata to the errors which
```
Comment (bartoszmajsak) [inline docs/modules/ROOT/pages/release_notes/v0.3.0.adoc:19]:
```suggestion
With https://github.com/emperror/emperror[emperror] we've got the ability to attach contextual metadata to the errors which 
```

**Review by bartoszmajsak — APPROVED** [review]
gifnews-pizza-baby-quXrRCpPUXt8k

**Comment by alien-ike** [comment]
#### ⚠ Failed generating changelog

#### Found unlabeled PRs

* https://github.com/maistra/istio-workspace/pull/826

**Comment by alien-ike** [comment]
🎇 🎉 🎆 New version has been shipped! 🚀🚀🚀

**Pull request merged**